test-commons/pom.xml

```diff
@@ -49,6 +49,12 @@
 			<artifactId>mockito-core</artifactId>
 			<version>2.12.0</version>
 		</dependency>
+
+		<dependency>
+			<groupId>uk.org.lidalia</groupId>
+			<artifactId>slf4j-test</artifactId>
+			<version>1.2.0</version>
+		</dependency>
```

Comment:
There are multiple errors during the test of our ReST API and our command line sub-projects stating...
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/Users/halprin/.m2/repository/ch/qos/logback/logback-classic/1.2.3/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/halprin/.m2/repository/uk/org/lidalia/slf4j-test/1.2.0/slf4j-test-1.2.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
```

That's because this `uk.org.lidalia:slf4j-test` is not marked for test and therefore conflicts with logback that we are using during normal use.  It seems we are getting lucky and lockback is being chosen by slf4j in normal usage, but I feel it could easily switch at some point and we would loose logging.

Reply:
test-commons is marked for test. The problem is the other loggers are also being included at compile time. I'll take a look in a sec.

Reply:
Should be fixed